packages/components/src/Button/Button.css

```
@@ -16,22 +16,24 @@
   border: var(--border-base) solid var(--button--color-variation);
   border-radius: var(--radius-base);
   color: var(--button--color-variation);
-  fill: var(--button--color-variation);
   -webkit-font-smoothing: antialiased;
+  letter-spacing: 0.2px;
```

We should use the value from the [design package](https://github.com/GetJobber/atlantis/blob/15f604cc51ee6881a5ed9c1be01081c76cf5ce6e/packages/design/src/typography.css#L3) here
```suggestion
  letter-spacing: var(--typography--letterSpacing-loose);
```